README.md

```diff
@@ -2,6 +2,15 @@
 
 This repository hosts the CSI KubeVirt driver and all of its build and dependent configuration files to deploy the driver.
 
+This CSI driver is made for a tenant cluster deployed on top of kubevirt VMs, and enables it to get its persistent data  
```

Comment:
on top of *openshift by creating* kubevirt VMs

Reply:
openshift is specific and I intentionally omitted that part, cause it can be any cluster. 
It is not clear enough?

Reply:
It's clear to me :)